index.js

```diff
@@ -107,34 +103,75 @@ async function init() {
 
 	updateStatus('Retrieving directory info…');
 
-	await validateInput(repo);
+	const repoMetadata = await validateInput(`${user}/${repo}`);
```

Comment:
Perhaps:

```suggestion
	const {private} = await fetchRepoInfo(user, repo);
```

Reply:
I considered this, but just returning that private boolean from the `validateInput` function seems even more weird to me. It should be fine if we find an appropriate name for the function.

Reply:
My suggestion includes the right name. The function will return the whole info, but we just destructure the `private` property.

Reply:
Gosh, I'm totally semi-blind today. Sorry for straining your nerves. 😅

Reply:
No worries, thank you for sorting this out ^_^